documentation/release_notes.markdown

```diff
@@ -12,6 +12,18 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 For release notes on versions of Puppet Server prior to Puppet Server 5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.8/release_notes.html).
 
+## Puppet Server 5.1.4
+
+Released September 6, 2017.
```

Comment:
This date seems wrong.

Reply:
@khussey Can you confirm the release date? The last I was told, it would be ready to ship Nov. 3 for a Nov. 6 ship date. The Final Daze doc says we ship Nov. 6. Confluence still has the Nov. 7 release date.